integration/presto/src/test/prestodb/org/apache/carbondata/presto/server/PrestoTestUtil.scala

```diff
@@ -114,4 +114,60 @@ object PrestoTestUtil {
       }
     }
   }
+
+  // this method depends on prestodb jdbc PrestoArray class
+  def validateArrayOfPrimitiveTypeDataWithLocalDict(actualResult: List[Map[String, Any]],
```

Comment:
Add it for prestosql also and compile for both prestodb and prestosql and run the testcase locally once with both the profile

Reply:
done